cmd/fyne_demo/screens/widget.go

```diff
@@ -185,9 +185,9 @@ func makeInputTab() fyne.CanvasObject {
 	selectEntry.PlaceHolder = "Type or select"
 	disabledCheck := widget.NewCheck("Disabled check", func(bool) {})
 	disabledCheck.Disable()
-	radio := widget.NewRadio([]string{"Radio Item 1", "Radio Item 2"}, func(s string) { fmt.Println("selected", s) })
+	radio := widget.NewRadioGroup([]string{"RadioGroup Item 1", "RadioGroup Item 2"}, func(s string) { fmt.Println("selected", s) })
```

Comment:
Should these strings not remain "Radio Item x"?

Reply:
Oops … yes!